lib/matplotlib/figure.py

```diff
@@ -764,7 +764,8 @@ def _add_axes_internal(self, ax, key):
         return ax
 
     def subplots(self, nrows=1, ncols=1, *, sharex=False, sharey=False,
-                 squeeze=True, subplot_kw=None, gridspec_kw=None):
+                 squeeze=True, subplot_kw=None, gridspec_kw=None,
+                 height_ratios=None, width_ratios=None):
```

Comment:
```suggestion
                 squeeze=True, height_ratios=None, width_ratios=None,
                 subplot_kw=None, gridspec_kw=None):
```
I suggest to put these first as they are more likely to be needed than the _kw params.